README.md

```diff
@@ -42,6 +42,7 @@ Config properties:
   response_type, // string; // Tells the authorization server which grant to execute.
   scope, // string[]; // A list of permissions that the application requires.
   redirectUri, // string; // Holds a URL. A successful response from this endpoint results in a redirect to this URL.
+  continueTo, // string; // [optional] Parameters added as `continue` appended to the `redirectUri`.
```

Comment:
This is not true, `continue` parameter doesn't not do anything special, it just pass the string around the server requests.